src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -160,91 +204,118 @@ public FormulaValue GetResult(IRContext irContext)
         private class MinDateAgg : IAggregator
         {
             protected DateTime _minValueDT = DateTime.MaxValue;
+            protected static readonly DateTime _defaultDT = new DateTime(1900, 1, 1);
 
             public void Apply(FormulaValue value)
-            {
-                if (value is BlankValue)
-                {
```

Comment:
>new DateTime(1900, 1, 1) [](http://example.com/codeflow?start=60&length=24)

This literal should only appear once. It could be a static at the top of this class,  or even a internal static on DateType

Reply:
Correct, fixed